api/src/test/java/org/pmiops/workbench/billing/BillingProjectBufferServiceTest.java

```diff
@@ -158,6 +160,7 @@ public void fillBuffer() {
                 .findByFireCloudProjectName(billingProjectName)
                 .getStatusEnum())
         .isEqualTo(BufferEntryStatus.CREATING);
+    verify(mockFireCloudService, never()).addProjectToServicePerimeter(anyString(), anyString());
```

Comment:
regardless of feature flags.

it may direct Terra to add the projects to the perimeter when creating the buffer.  That's tested elsewhere.